V2/tests/api/azure/time_entry_azure_endpoints_test.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import azure.functions as func
+
+import time_tracker.time_entries._application._time_entries as azure_time_entries
+
+TIME_ENTRY_URL = "/api/time-entries/"
+
+
+def test__time_entry_azure_endpoint__creates_an_time_entry__when_time_entry_has_all_attributes(
+    create_fake_database, time_entry_factory, activity_factory, insert_activity
+):
+    db = create_fake_database
+    inserted_activity = insert_activity(activity_factory(), db)
+    time_entry_body = time_entry_factory(activity_id=inserted_activity.id, technologies="[jira,sql]").__dict__
+
+    azure_time_entries._create_time_entry._DATABASE = db
```

Comment:
This weird stuff we have to do in our tests is happening because we should instantiate _DATABASE inside a function